kubeadm/scripts/Install-Containerd.ps1

```diff
@@ -137,27 +137,6 @@ $prefixLength = (Get-NetIPAddress -InterfaceAlias $netAdapterName -AddressFamily
 
 $subnet = CalculateSubNet -gateway $gateway -prefixLength $prefixLength
 
-@"
```

Comment:
we should clean up the line above this as well: 

```
Write-Output "Creating network config for nat network"
$gateway = (Get-NetIPAddress -InterfaceAlias $netAdapterName -AddressFamily IPv4).IPAddress
$prefixLength = (Get-NetIPAddress -InterfaceAlias $netAdapterName -AddressFamily IPv4).PrefixLength
```

Reply:
why do we still calculat the subnet if we don't need the nat config anymore below?
Am I missing something or is this code obsolet?